src/pages/Main/AddMovieButton.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+function AddMovieButton(props) {
```

Comment:
```suggestion
function AddMovieButton({ display, onAddMovieListClick }) {
```
함수 파라미터 자리에서 직접 구조분해 할당 해줄 수 있습니다

Reply:
넵. 디테일 페이지에서도 나왔던 문제군요.
반영하겠습니다.